### PR TITLE
BaSP-TG-30: Test Admin: get, getById, post

### PR DESCRIPTION
**Pull request by silveriac**

### CODE REVIEW

File: src/test/admins.test.js [hunk 1]
```diff
@@ -1 +1,73 @@
...
+  dni: '27378785',
+  phone: '1166684785',
+  location: 'Buenos Aires',
+};
+
+describe('GET /admin/:id', () => {
+  test('everything correct: return status code 200', async () => {
+    const res = await request(app).get(`/admins/${adminId}`);
+    expect(res.status).toBe(200);
+  });
```
Comment (MaicolFreire001) [inline src/test/admins.test.js:29]: We could add expects for error boolean and/or message

File: src/test/admins.test.js [hunk 2]
```diff
@@ -1 +1,73 @@
...
+    expect(res.body.error).toBeTruthy();
+  });
+});
+
+describe('POST /admin/:id', () => {
+  test('everything correct: return status code 201', async () => {
+    const res = await request(app).post('/admins/').send(mockedAdmin);
+    // eslint-disable-next-line no-underscore-dangle
+    newId = res.body.data._id;
+    expect(res.status).toBe(201);
```
Comment (MaicolFreire001) [inline src/test/admins.test.js:49]: We could add expects for error boolean and/or message

File: src/test/admins.test.js [hunk 3]
```diff
@@ -1 +1,73 @@
...
+    expect(res.body.message[0].message).toBe('first name required');
+    expect(res.body.error).toBeTruthy();
+  });
+});
+
+describe('GET /admin/', () => { // leave GET at the end when merging (second test deletes only item on DB)
+  test('everything correct: return status code 200', async () => {
+    const res = await request(app).get('/admins/');
+    expect(res.status).toBe(200);
+  });
```
Comment (MaicolFreire001) [inline src/test/admins.test.js:64]: We could add expects for error boolean and/or message

**Review by MaicolFreire001 — CHANGES_REQUESTED** [review]

File: src/test/admins.test.js [hunk 4]
```diff
@@ -1 +1,74 @@
...
+  dni: '27378785',
+  phone: '1166684785',
+  location: 'Buenos Aires',
+};
+
+describe('GET /admin/:id', () => {
+  test('everything correct: return status code 200', async () => {
+    const res = await request(app).get(`/admins/${adminId}`);
+    expect(res.status).toBe(200);
+    expect(res.body.error).toBeFalsy();
```
Comment (nicolasblasco) [inline src/test/admins.test.js:27]: We should also test the message of the body

File: src/test/admins.test.js [hunk 5]
```diff
@@ -1 +1,74 @@
...
+  });
+});
+
+describe('POST /admin/:id', () => {
+  test('everything correct: return status code 201', async () => {
+    const res = await request(app).post('/admins/').send(mockedAdmin);
+    // eslint-disable-next-line no-underscore-dangle
+    newId = res.body.data._id;
+    expect(res.status).toBe(201);
+    expect(res.body.error).toBeFalsy();
```
Comment (nicolasblasco) [inline src/test/admins.test.js:49]: We should also test the message of the body

File: src/test/admins.test.js [hunk 6]
```diff
@@ -1 +1,74 @@
...
+    expect(res.body.message[0].message).toBe('first name required');
+    expect(res.body.error).toBeTruthy();
+  });
+});
+
+describe('GET /admin/', () => { // leave GET at the end when merging (second test deletes only item on DB)
+  test('everything correct: return status code 200', async () => {
+    const res = await request(app).get('/admins/');
+    expect(res.status).toBe(200);
+    expect(res.body.error).toBeFalsy();
```
Comment (nicolasblasco) [inline src/test/admins.test.js:64]: We should also test the message and data of the body

**Review by nicolasblasco — CHANGES_REQUESTED** [review]

**Review by nicolasblasco — APPROVED** [review]

**Pull request merged**